Assets/SEE/Game/UI3DProperties.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+
+namespace SEE.UI3D
+{
+    /// <summary>
+    /// Properties of 3d ui-elements.
```

Comment:
```suggestion
    /// Properties of 3D UI elements.
```

(Other instances in this class should be renamed to `UI element` too.